### PR TITLE
Docs sorting

### PR DESCRIPTION
**Pull request by alexfornuto**
## Summary

- Sorts the Transmission guide alphabetically in side bar and guide index.
- Renames "code-server" to "Visual Studio Code Server" for better clarity to the unfamiliar, as well as easier alphabetizing.
- Capitalizes Pomerium throughout guide index.

## Checklist

- NA reference any related issues
- [X] updated docs
- NA updated unit tests
- NA updated UPGRADING.md
- [X] add appropriate tag (`improvement` / `bug` / etc)
- [ ] ready for review

### CODE REVIEW

File: docs/guides/code-server.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: code-server
+title: Visual Studio Code Server
```
Comment (desimone) [inline docs/guides/code-server.md:2]: I think `code-server` is the correct name for the project now. 

\cc @jsjoeio

Comment (jsjoeio) [reply docs/guides/code-server.md:2]: yes, that's right!

Comment (alexfornuto) [reply docs/guides/code-server.md:2]: Yea, I was mostly using this terminology for those unfamiliar with code-server, who may not immediately realize it's related to VSCode.

Comment (jsjoeio) [reply docs/guides/code-server.md:2]: well you could always put "code-server — VSCode in the browser" but code-server should be used for the name

**Review by desimone — APPROVED** [review]

File: docs/guides/readme.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
+- The [local OIDC](./local-oidc.md) guide demonstrates how Pomerium can be used with local OIDC server for dev/testing.
+- The [mTLS](./mtls.md) guide demonstrates how Pomerium can be used to add mutual authentication using client certificates and a custom certificate authority.
+- The [TiddlyWiki](./tiddlywiki.md) guide demonstrates how Pomerium can be used to add authentication and authorization to web application using authenticated header.
+- The [Transmission](./transmission.md) guide demonstrates how Pomerium can act as an authentication and authorization proxy for your Transmission daemon's RPC interface, which only provides unencrypted HTTP auth out of the box.
+- The [Visual Studio Code](./code-server.md) guide demonstrates how Pomerium can be used to add access control to third-party applications that don't ship with [fine-grained access control](https://github.com/cdr/code-server/issues/905).
```
Comment (jsjoeio) [inline docs/guides/readme.md:15]: Shouldn't this say code-server?
```suggestion
- The [code-server](./code-server.md) guide demonstrates how Pomerium can be used to add access control to third-party applications that don't ship with [fine-grained access control](https://github.com/cdr/code-server/issues/905).
```

File: docs/guides/code-server.md (hunk 1 above)
Comment (jsjoeio) [inline docs/guides/code-server.md:2]:
```suggestion
title: code-server
```

**Review by jsjoeio — CHANGES_REQUESTED** [review]

**Pull request merged**